datafusion/physical-plan/src/joins/hash_join_utils.rs

```diff
@@ -61,44 +61,45 @@ use hashbrown::HashSet;
 ///
 /// ``` text
 /// See the example below:
-/// Insert (1,1)
+///
+/// Insert (10,1)            <-- insert hash value 10 with row index 1
```

Comment:
I found the overloaded use of `1` as both a hash value and an index VERY confusing. Therefore I propose to use different values for the hash value and the index values